PyInstaller/hooks/hook-django.py

```diff
@@ -80,7 +82,8 @@
             datas.append((f, os.path.join(mod_name, bundle_dir)))
 
     # Include data files from your Django project found in your django root package.
-    datas += collect_data_files(package_name)
+    # excludes files from the project dir itself (changed behavior of collect_data_files)
+    datas += collect_data_files(package_name, excludes=os.listdir())
```

Comment:
This shouldn't be needed. Excluding `os.listdir` should break the hook actually.

Reply:
For some reason, it adds all elements from the directory, not only the django itself (mentioned in the issue).
I created new django project from scratch, and indeed, exclude is not needed in that case - probably a bug in my build scripts, which I don't have time to resolve